chpl/pom.xml

```diff
@@ -10,7 +10,7 @@
   <modules>
     <module>chpl-api</module>
     <module>chpl-service</module>
-    <module>chpl-auth</module>
+    <!-- <module>chpl-auth</module> -->
```

Comment:
Recommend deleting this instead of leaving commented

Reply:
Addressed